sdk/personalizer/Azure.AI.Personalizer/src/Models/DecisionContext.cs

```diff
@@ -21,11 +25,11 @@ public DecisionContext()
         /// <param name="rankableActions"> Rankable actions </param>
         public DecisionContext(IEnumerable<object> contextFeatures, List<PersonalizerRankableAction> rankableActions)
```

Comment:
The PR mostly looks good, but the idea here was not to ship a public API that used simply `object`, which is what gets passed to this constructor. That should be a list of `BinaryData`. You haven't GA'd yet, so it's acceptable to break the API - preferred, in fact, to make sure we ship good APIs. /cc @tg-msft

Reply:
changed from IEnumerable<object> to IList<BinaryData>